aries-site/src/pages/_app.js

```diff
@@ -167,10 +167,6 @@ function App({ Component, pageProps, router }) {
             }
           }
         }
-        window.localStorage.setItem(
```

Comment:
@beahackman where do we updated the local storage key for this now? Are we still saving the last 30 days in local storage? If so, it does seem like we need to set it somewhere

Reply:
I might be missing where this logic is occuring though.

Reply:
Yea! So basically "update-history" was a local storage copy of what is now just held in contentHistory (the variable I pass through context.) Before this commit, I had both update-history and contentHistory (for some weird inefficient reasons haha.) In the commit "removing local storage of update history" I basically consolidated everything down to just use context, so that "update-history" wouldn't be necessary in any function. However, I just forgot to get rid of the place where I set it (now that nothing uses it anymore.)
I still set/hold the time-tokens in local storage like on app.js 176 & 197, but those just have the page names and times.

Reply:
Gotcha! Thank you for clarifying!